guides/micronaut-microservices-services-discover-eureka/bookrecommendation/java/src/main/java/example/micronaut/BookCatalogueClient.java

```diff
@@ -13,14 +13,14 @@
 @Recoverable(api = BookCatalogueOperations.class)
 //end::harcoded[]
 */
-//tag::eureka[]
+//tag::consul[]
```

Comment:
Why are we renaming the asciidoc tag in the eureka guide from `eureka` to  `consul`?